assets/lang/sklmessages_pl_PL.properties

```diff
@@ -73,7 +73,7 @@ editor.compatibility.tooltip=Skiny nie działają w tym trybie, ale niektóre pa
 editor.exe=Plik wykonywalny
 editor.jvm=Argumenty JVM
 editor.alert.title=Potwierdzenie profilu
-editor.alert.text=Czy na pewno chcesz usunąć ten profil?
+editor.alert.text=Czy na pewno chcesz usunąć ten profil? Tego nie da się odwrócić!
```

Comment:
Tego nie ma w stringu źródłowym, więc tu też tego nie powinno być

Reply:
Myślę że można by było to dodać do en_US

Reply:
Moim zdaniem mogłoby zostać, wygląda na całkiem pomocne. Co do treści, to "odwrócić" w tej sytuacji nie wygląda zbyt dobrze. Skłaniałbym się raczej do czegoś w stylu "Tego nie da się cofnąć!" lub lepiej – "Tego nie można cofnąć!".

Reply:
Moim zdaniem mogłoby zostać, wygląda na całkiem pomocne. Co do treści, to "odwrócić" w tej sytuacji nie wygląda zbyt dobrze. Skłaniałbym się raczej do czegoś w stylu "Tego nie da się cofnąć!" lub lepiej – "Tego nie można cofnąć!".

Reply:
To kwestia gustu, co brzmi lepiej. Może niech @CodeIndexer zdecyduje, który tekst mu się bardziej podoba – "Tego nie da się cofnąć!" czy "Tego nie można cofnąć!" – i wprowadzi poprawkę. Bo nie da się ukryć, że obecne zdanie nie brzmi najlepiej, więc dodatkowy fix i tak jest potrzebny.

Reply:
Osobiście bardziej mi się podoba **"Tego nie można cofnąć!"**. Za chwilkę poprawię.